### ISSUE TITLE
Composite form fields should be grouped

### ISSUE THREAD

**Issue by jeanniefu** [civiform/civiform#2618]
Feedback from a11y experts during an a11y audit.

Severity (Critical/Serious/Moderate/Good to have): Serious

- Form fields like checkboxes and radio buttons **should** be grouped. The a11y team recommended we use fieldset and legend (see **first** resource link below for an example).
- Address **can** be grouped (see **second** resource link below for an example).
- In the future, if we add a phone number field, we should group these fields (see **third** resource link below for an example)
- Should also group name (We currently have first, middle, last as separate fields)

Resources: 
* https://www.w3.org/WAI/tutorials/forms/grouping/
* https://www.w3.org/TR/WCAG20-TECHS/H71.html
* https://www.w3.org/TR/WCAG20-TECHS/ARIA17.html

**Comment by jeanniefu** [comment]
For grouping composite fields like address and name (e.g. these have multiple input fields associated with one question), we should attach the question level description and errors to the fieldset.

**Comment by jeanniefu** [comment]
Reminder to myself: We should also group the radio buttons for the language selector page (note that that goes through a different code flow than an application question radio button).
